projects/views.py

```diff
@@ -530,6 +572,18 @@ def view_project(request, project_id):
             user["email"] = portal_user.email
             user["first_name"] = portal_user.first_name
             user["last_name"] = portal_user.last_name
+            try:
+                su_budget = ChargeBudget.objects.get(
+                    user=portal_user, project=budget_project
+                )
+            except ChargeBudget.DoesNotExist:
+                su_budget_value = current_allocation_su_allocated
+            else:
+                su_budget_value = su_budget.su_budget
+            user["su_budget"] = int(su_budget_value)
+            user["su_used"] = ChargeBudget(
```

Comment:
It's a little confusing that this creates a ChargeBudget object, since it's not actually getting it from the database? Insteade maybe `current_usage` should be a static/util method somewhere?